resources/puppetlabs/lein-ezbake/template/pe/ext/debian/control.erb

```diff
@@ -2,7 +2,10 @@ Source: <%= EZBake::Config[:project] %>
 Section: utils
 Priority: optional
 Maintainer: Puppet Labs <info@puppetlabs.com>
-Build-Depends: debhelper (>= 7.0.0~), cdbs, bc, mawk, lsb-release, puppet-agent
+Build-Depends: debhelper (>= 7.0.0~), cdbs, bc, mawk, lsb-release, puppet-agent<%=
+    if !EZBake::Config[:debian][:additional_build_dependencies].empty?
```

Comment:
The way these are handled could stand to be refactored, but this will work. Can we make a note to clean up the way this list is constructed later on? Or is that something we can fix relatively quickly in a followup `(maint)` PR?